syft/generic/pointers/pointer_tensor.py

```diff
@@ -404,12 +413,17 @@ def share(self, *args, **kwargs):
             A pointer to an AdditiveSharingTensor
         """
 
-        # Send the command
-        command = ("share", self, args, kwargs)
+        ptr_id = self.id_at_location
+        source = self.owner.id
+        _kwargs = {}
+        message = TensorCommandMessage.communication(
+            ptr_id, "share", source, [self.location.id], args, _kwargs
```

Comment:
Maybe I'm wrong but I think args for share are really the location where you want to send shares... So maybe this should go into `location` instead of `[self.location.id]` which is the location of the tensor to share (so basically the source of the communication action we want to perform)

Reply:
Yeah, it probably should, but for this PR I'd be satisfied with just adding names and fixing the things that the extra field breaks. (We can keep refactoring from there of course, but trying to keep the scope of each PR limited.)

Reply:
@Jasopaum you are right, the args are just a list of where to SMPC "share" the tensor. this also means we can remove `args` I think, which is great news... I have updated the code to reflect that.

I am using send_msg instead of send_command so I think location is self.owner... 
(because the doc string for `send_msg => args: location = A BaseWorker instance that lets you provide the destination to send the message.`)
plz let me know if this understanding is incorrect 

also destination is where a communication action is being sent so I think that is the ids of the workers we can pull out of `args`... also let me know if this is incorrect